From: Michal Hocko <mhocko@suse.cz>
Subject: Hide new tk_rebind_retry from kABI checker
Patch-mainline: no
References: bnc#702013

rpc_task.tk_rebind_retry added by the previous patch doesn't alter the
structure size nor its layout. So no external user of this structure
will notice. We have to hide it from the kABI checker anyway because
the structure has changed from its POV.

Signed-off-by: Michal Hocko <mhocko@suse.cz>
Index: linux-2.6.32-SLE11-SP1/include/linux/sunrpc/sched.h
===================================================================
--- linux-2.6.32-SLE11-SP1.orig/include/linux/sunrpc/sched.h
+++ linux-2.6.32-SLE11-SP1/include/linux/sunrpc/sched.h
@@ -55,7 +55,9 @@ struct rpc_task {
 	struct rpc_message	tk_msg;		/* RPC call info */
 	__u8			tk_garb_retry;
 	__u8			tk_cred_retry;
+#ifndef  __GENKSYMS__
 	__u8			tk_rebind_retry;
+#endif
 
 	/*
 	 * callback	to be executed after waking up
